src/pages/cooking/cooking.html

```diff
@@ -24,14 +24,13 @@
       </ion-item-options>
     </ion-item-sliding>
   </ion-list>
-  <button ion-button block
-    (click)="doneCooking()">Done Cooking!</button>
 </ion-content>
 
 <ion-footer>
   <ion-toolbar>
     <p class="center">
       <strong>Time Elapsed: </strong>{{ticks}} mins | <strong>Time Remaining: </strong> {{totalTime - ticks}} mins
+      <button ion-button block (click)="doneCooking()">Done Cooking!</button>
```

Comment:
I would vote to leave it there, regardless of whether or not all the steps are marked completed just because you could get to the end of your cooking and might have five "Take out of the oven!" steps and swiping "complete" on all those instead of just hitting one button could be tedious. Also for us, as far as testing goes, it's way easier to just hit a button and not bother marking all steps as completed first. But I'll go ahead and change the color :+1: